vars/buildPlugin.groovy

```diff
@@ -53,20 +51,17 @@ def call() {
                 }
             }
 
-            node('linux-dev') {
-                stage ('Publish Test Results') {
-                    sh 'rm -rf *'
-                    unstash 'test_results'
-                    if ( params.run_integration_tests ) {
-                        unstash 'integration_test_results'
-                    }
+            node('master') {
+               stage ('Publish Test Results') {
+                    dir('tests') {
+                        unstash 'test_results'
+                        if ( params.run_integration_tests ) {
+                            unstash 'integration_test_results'
+                        }
 
-                    junit healthScaleFactor: 100, testResults: testResultFilePatterns.surefire + ', ' + testResultFilePatterns.failsafe
-                    step([$class: 'FindBugsPublisher', canComputeNew: false, defaultEncoding: '', excludePattern: '', healthy: '', includePattern: '', pattern: testResultFilePatterns.findBugs, unHealthy: ''])
-                }
-                stage('Upload Bits') {
```

Comment:
Why remove this stage?

Reply:
We have local archives and Artifactory repo, so this stage isn't necessary any more